### PR TITLE
Simplify docs folder structure, move to Black GitHub Actions

### PR DESCRIPTION
**Pull request by danielhuppmann**
Two small clean-ups because Stickler terminated its service and to keep consistency with the pyam package.

### CODE REVIEW

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -10,20 +10,33 @@ on:
     branches: [ '**' ]
 
 jobs:
-  build:
+  pytest:
+    strategy:
+      matrix:
```
Comment (phackstock) [inline .github/workflows/build-docs.yml:15]: Do we need a matrix for a single OS version with a single python version?

File: docs/make.bat [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
-
-%SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
-goto end
-
-:help
-%SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
-
-:end
-popd
+REM This file was generated with sphinx-quickstart version 3.5.0
```
Comment (phackstock) [inline docs/make.bat:1]: Do we actually need this file? From what I understand `.bat` files are for use on Windows and I don't think anyone is building the docs on Windows.

**Review by phackstock — COMMENTED** [review]
Looks good, just two minor comments

File: .github/workflows/build-docs.yml (hunk 1 above)
Comment (danielhuppmann) [reply .github/workflows/build-docs.yml]: Simplified.

File: docs/make.bat (hunk 2 above)
Comment (danielhuppmann) [reply docs/make.bat:1]: I'd keep it as long as there is no effort required to maintain it...

**Pull request merged**